lib/streamlit/watcher/LocalSourcesWatcher.py

```diff
@@ -107,9 +108,20 @@ def close(self):
         self._is_closed = True
 
     def _register_watcher(self, filepath, module_name):
-        wm = WatchedModule(
-            watcher=FileWatcher(filepath, self.on_file_changed), module_name=module_name
-        )
+        if compatibility.is_running_py3():
+            ErrorType = PermissionError
+        else:
+            ErrorType = OSError
```

Comment:
Nit, personal liking or coding style ignorance...
```
ErrorType = PermissionError if compatibility.is_running_py3() else OSError
```

Reply:
That's too long, so I'd have to use parentheses and wrap the line. At which point the `if/else` block I used is cleaner.

Reply:
Ok.